dfastmi/io/ConfigFileOperations.py

```diff
@@ -153,15 +153,16 @@ def _update_to_relative_path(
             Configuration for the D-FAST Morphological Impact analysis with only relative paths.
         """
         absolute_path = config.get(section, key, fallback="")
-        if len(absolute_path) == 0:
-            absolute_path = rootdir
-
-        absolute_path_converted_to_relative_path = str(
-            os.path.relpath(absolute_path, rootdir)
-        )
 
-        if absolute_path_converted_to_relative_path == ".":
+        if len(absolute_path) == 0:
```

Comment:
Why was this updated, was something not working?

Reply:
Behaviour with empty paths is currently unsure.
@hrajagers will have a look at this.

Reply:
The main issue that I encountered was that the absolute_path to relative_path worked only when the path and the reference are on the same drive. It would crash the code when trying to come up with a relative path to a file on a different drive. The new code falls back to using the absolute path when that happens.

Furthermore, if the absolute_path (this is the path shown in the GUI) is empty, the relative path should also be empty. Otherwise, fields that haven't been entered will suddenly point to the rootdir. This was fixed by the line that converts relative paths "." to empty string again at the end, but this would not allow you to use the rootdir/current folder as the folder for output.

Reply:
We will leave it as is.